### ISSUE TITLE
saveUserTags results in "no tag found"

### ISSUE THREAD

**Issue by leecrossley** [Donky-Network/DonkySDK-iOS-Modular#1]
I registered a new user, then created 2 tags in Donky Control (test1, test2).

When I call `usersTags` I get the follow log output:

```
Request api/registration/user/tags successful, response data = (
        {
        isSelected = 0;
        value = test1;
    },
        {
        isSelected = 0;
        value = test2;
    }
)
```

but I'm unable to `saveUserTags`, although the data I'm sending looks to be correct (even the success block is called):

<img src="http://oi58.tinypic.com/nccqkz.jpg"/>

I can see the following in the output log:
```
[DNUserDetails toggleTag:isSelected:] [line 158] ERROR: No tag found for: test2
[DNUserDetails toggleTag:isSelected:] [line 158] ERROR: No tag found for: test1
```

Am I missing something or is there an issue? (v2.4.4.4)

Help is much appreciated.

**Comment by chriswatson00** [comment]
Hi Lee, 

Are you are using the method saveUserTags: inside the DNUserDetails object ?

**Comment by leecrossley** [comment]
I'm calling DNAccountController saveUserTags

**Comment by chriswatson00** [comment]
Hi, 
I've just run a test, there is indeed an issue with the tags being saved locally. I have corrected this error can will commit the changes shortly. Thank you for letting us know.

**Comment by chriswatson00** [comment]
Are you using cocoapods?

**Comment by leecrossley** [comment]
I'm using the source directly from git, not cocoapods - once you push to Github I can easily upgrade at this end.

**Comment by chriswatson00** [comment]
Ok. Just running through the tests etc. Will drop you a line when it's pushed.

**Comment by leecrossley** [comment]
Thanks for the quick fix, if you could reference the commit here that'd be great :)

**Comment by chriswatson00** [comment]
Yep I will do.

**Comment by chriswatson00** [comment]
Resolved in release: [2.5.4.3](https://github.com/Donky-Network/DonkySDK-iOS-Modular/releases/tag/v4.5.4.4)

**Comment by leecrossley** [comment]
I can confirm the tags are now persisted locally - thanks again.
